### PR TITLE
[stable/nginx-ingress] Allow hostNetwork when PSP is enabled

### PR DESCRIPTION
**Pull request by tjakobsson**
#### What this PR does / why we need it:

This PR fixes so PodSecurityPolicy allows using hostNetwork if both are enabled.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped

### CODE REVIEW

File: stable/nginx-ingress/templates/podsecuritypolicy.yaml [hunk 1]
```diff
@@ -20,7 +20,11 @@ spec:
     #- 'projected'
     - 'secret'
     #- 'downwardAPI'
+  {{- if .Values.controller.hostNetwork }}
+  hostNetwork: true
```
Comment (cpanato) [inline stable/nginx-ingress/templates/podsecuritypolicy.yaml]: you can do 
```
hostNetwork: {{ .Values.controller.hostNetwork | default false }}
```

and then remove the if statement

Comment (unguiculus) [reply stable/nginx-ingress/templates/podsecuritypolicy.yaml]: It should be `hostNetwork: {{ .Values.controller.hostNetwork }}`. The default value comes from `values.yaml`.

**Review by unguiculus — APPROVED** [review]

**Review by unguiculus — CHANGES_REQUESTED** [review]
Hit the wrong radio button. Didn't mean to approve.

**Comment by unguiculus** [comment]
Closing in favor of #10776 which implements this as suggested. Thanks.

**Pull request closed without merging**